Watchman.Integrations/Images/ImagesService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Watchman.Integrations.Images
+{
+    public class ImagesService
+    {
+        public IEnumerable<KeyValuePair<string,Stream>> GetImagesFromResources(Func<string, bool> selector = null)
+        {
+            var imagesProperty = typeof(Images).GetProperties().Where(x => x.PropertyType.Name == "Byte[]");
+
+            foreach (var image in imagesProperty)
+            {
+                if (selector == null || selector.Invoke(image.Name))
+                {
+                    var imageAsByteArray = (byte[])image.GetValue(image);
+                    yield return new KeyValuePair<string, Stream>(image.Name, new MemoryStream(imageAsByteArray));
```

Comment:
na tym etapie mogłoby być dodawanie rozszerzenia do nazwy
hmm
może z tych byte[] jest spobób na pobranie MIME
a jak nie, to na sztywno można dorzucać .png 
w przyszłości można będzie przebudować tą klase i zwracane pliki to powinny być już pliki z nazwami z rozszerzeniem